tests/venv/postBuild/README.rst

```diff
@@ -5,3 +5,10 @@ It is possible to run scripts after you've built the environment specified in
 your other files. This could be used to, for example, download data or run
 some configuration scripts. For example, this will download and install a
 Jupyter extension.
+
+.. note::
+
+   This file needs to be executable in order to work with ``repo2docker``. The
+   easiest way to do this is to run the following command with ``git``::
+
+       git update-index --chmod=+x postBuild

```

Comment:
Is... this the easiest way to do it? Isn't that just 'chmod +x postBuild'? What does update-index do? I've never encountered it before...

Reply:
Def not the easiest, but I don't think chmod exists in windows, so this is a command that (I think) should work on any platform. I could expand the text to say this

Reply:
Please do! I'd prefer we just said 'on OS X and Linux, do `chmod +x postBuild` and on Windows do git <blah>' rather than just have one blanket unexplained command.

Reply:
Sounds good. I guess I just think of this command as the chmod version of what "git mv" is to "mv"